### PR TITLE
public map: fix buttons and caret

### LINKED ISSUE #6490: Mising styles in sign up buttons
In public pages
image

### PR DESCRIPTION
**Pull request by matallo**
close #6490

screen shot 2016-01-29 at 18 44 01

CR @piensaenpixel @xavijam

### CODE REVIEW

**Comment by xavijam** [comment]
caretDown is a new icon from CartoAssets or it is already there?

**Comment by matallo** [comment]
Checked and it was already there, it's in ded02

**Comment by matallo** [comment]
so, is it a :+1: ? :)

File: lib/build/files/css_files.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ module.exports = css_files = {
   ],
 
   public_dashboard: [
-    '<%= assets_dir %>/stylesheets/public_dashboard/**/*.css'
+    '<%= assets_dir %>/stylesheets/public_dashboard/**/*.css',
+    '<%= assets_dir %>/stylesheets/public_map/public_map_buttons.css',
```
Comment (xavijam) [inline lib/build/files/css_files.js]: Why do you add them to the public dashboard?

Comment (matallo) [inline lib/build/files/css_files.js]: that is the view where those styles were missing

**Comment by xavijam** [comment]
I don't know if we should move those buttons to a public common view, but in any case, :+1:. Let's face that problem in another place.

**Pull request merged**